config/locales/en/documents/show.yml

```diff
@@ -69,3 +69,10 @@ en:
           chosen: "Image “%{file}” has been selected as the lead image"
           removed: "Image “%{file}” has been deselected. Document will now use the default lead image"
           deleted: "Image “%{file}” has been deleted. Document wil now use the default lead image"
+        topics_updated: Topics updated on all editions
```

Comment:
Is this content designer approved? Feels like it could be a point of confusion.

Reply:
Yup - @benhazell has commented on the card and we've updated 👍 .